src/annotator-custom-editor.js

```diff
@@ -0,0 +1,27 @@
+Annotator.Plugin.CustomEditor = class CustomEditor extends Annotator.Plugin{
+  pluginInit(){
+    this.annotator
+      .subscribe('beforeAnnotationCreated', annotation => {
```

Comment:
ok I see you did that here! I guess I'm getting confused about the file structure – so the code files are in `./src`, and these get transpiled into `./lib`–– still unclear as to purpose of `./tmp`